WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Delete WordPress/UIButton+Dismiss.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+
+extension UIButton {
+
+    private static var closeButtonImage: UIImage {
+        let fontForSystemImage = UIFont.systemFont(ofSize: Metrics.closeButtonRadius)
+        let configuration = UIImage.SymbolConfiguration(font: fontForSystemImage)
+
+        // fallback to the gridicon if for any reason the system image fails to render
+        return UIImage(systemName: Constants.closeButtonSystemName, withConfiguration: configuration) ??
+        UIImage.gridicon(.crossCircle, size: CGSize(width: Metrics.closeButtonRadius, height: Metrics.closeButtonRadius))
+    }
+
+    static func makeCloseButton() -> UIButton {
```

Comment:
My intention is to provide and easy API to create and use the "Dismiss Button". And this component is supposed to be used in any screen, not just migration screens.

The file `UIButton+Dismiss.swift` location is `/WordPress-to-Jetpack Migration/Delete WordPress/UIButton+Dismiss.swift`. Is there a better spot for reusable components?

Reply:
👋🏼 ! I'd say the current location is fine. 

When we do get cases where people would like to reuse the button, we can relocate it into a more feature-agnostic group such as `ViewRelated/Extensions` or `ViewRelated/Views`. If it's potentially reusable across apps, we can relocate it into the `WordPressUI` repository.